clisops/core/subset.py

```diff
@@ -1044,6 +1170,57 @@ def subset_time(
     return da.sel(time=slice(start_date, end_date))
 
 
+@check_start_end_levels
+def subset_level(
+    da: Union[xarray.DataArray, xarray.Dataset],
+    first_level: Optional[Union[int, float]] = None,
+    last_level: Optional[Union[int, float]] = None,
+) -> Union[xarray.DataArray, xarray.Dataset]:
+    """Subset input DataArray or Dataset based on first and last levels.
+    Return a subset of a DataArray or Dataset for levels falling within the provided bounds.
+
+    Parameters
+    ----------
+    da : Union[xarray.DataArray, xarray.Dataset]
+      Input data.
+    first_level : Optional[Union[int, float]]
+      First level of the subset.
```

Comment:
I suggest to clarify that this is not the level index, but the level value.

Reply:
I have updated the doc string to make it clearer.